src/components/AffectsTable/MultiFlawTracker.vue

```diff
@@ -0,0 +1,113 @@
+<script setup lang="ts">
+import { computed, ref } from 'vue';
+
+import { useMultiFlawTrackers } from '@/composables/useMultiFlawTrackers';
+
+import DropDownMenu from '@/widgets/DropDownMenu/DropDownMenu.vue';
+
+const {
+  actions: { addFlaw: addFlawAction, removeFlaw, validateId },
+  computed: { cveStreamCount },
+  state: { relatedAffects },
+} = useMultiFlawTrackers();
+
+const searchFilter = ref<string>('');
+
+const isSearchValid = computed(() => {
+  const trimmed = searchFilter.value.trim();
+  return trimmed.length > 0 && validateId(trimmed);
+});
+
+function addFlaw() {
+  const trimmed = searchFilter.value.trim();
+  if (trimmed && validateId(trimmed)) {
+    addFlawAction(trimmed);
+    searchFilter.value = '';
+  }
+}
+
+function handlePaste(event: ClipboardEvent) {
+  const pastedText = event.clipboardData?.getData('text') || '';
+  const entries = pastedText
+    .split(/[\n,\s]+/)
+    .map(entry => entry.trim())
+    .filter(entry => entry.length > 0);
+
+  if (entries.length <= 1) return;
+
+  event.preventDefault();
+
+  entries.forEach((entry) => {
+    if (validateId(entry)) {
+      addFlawAction(entry);
+    }
+  });
+
+  searchFilter.value = '';
+}
+</script>
+
+<template>
+  <DropDownMenu>
+    <template #trigger>
+      <button
+        type="button"
+        title="Open Tracker Manager"
+        class="btn btn-secondary text-nowrap"
+      >        <i class="bi-collection-fill"></i>
+      </button>
+    </template>
+    <template #content>
+      <ul class="list-unstyled">
+        <li class="dropdown-item bg-white">
+          <div class="input-group">
+            <input
+              v-model="searchFilter"
+              class="border border-info px-1 focus-ring focus-ring-info"
+              type="text"
+              placeholder="CVE or Flaw UUID"
+              @keypress.enter="isSearchValid && addFlaw()"
+              @paste="handlePaste"
+            />
+            <button
+              :disabled="!isSearchValid"
+              class="btn btn-info btn-sm font-white"
+              type="button"
+              @click="addFlaw()"
+            >Add</button>
```

Comment:
Minor nitpick for readability:

```suggestion
            >
               Add
            </button>
```